builtin/logical/pki/path_fetch.go

```diff
@@ -315,6 +333,79 @@ reply:
 	return
 }
 
+func (b *backend) readCertByAlias(ctx context.Context, req *logical.Request, alias string) ([]byte, error) {
```

Comment:
this seems to duplicate what is already in pathFetchRead: is that right?

Reply:
Yes; if we later refactor the JSON endpoints, we'll refactor `pathFetchRead` and likely use this logic instead of the logic already in it. 

I didn't really see value in updating `pathFetchRead` to take out this piece of logic (avoiding the duplication) for the time being, since the bigger refactor of JSON endpoints would probably impact how we structure it anyways.

That's my 2c. at any rate.